README.md

```diff
@@ -22,7 +22,11 @@ Then navigate to the `topicalguide` directory and optionally switch over to the
 
 ### 2. Install Dependencies
 
-Dependencies are listed in the `requirements.txt` file and can be easily installed with:
+First, we need to make sure numpy is installed with:
+
+    sudo pip install numpy
```

Comment:
I hesitate to recommend `sudo` installing any sort of package for any platform.  Too many years of problems down the road going this route.  I want to keep dev environments disposable, and a `sudo` installation breaks that.  I just noticed that the dev branch's README.md recommends `sudo`ing, so that's something I would like to change.

I have a few questions about the original issue, and I'll ask them on that issue.